src/operator/pad.cc

```diff
@@ -400,12 +400,65 @@ Operator *PadProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_shape,
 DMLC_REGISTER_PARAMETER(PadParam);
 
 MXNET_REGISTER_OP_PROPERTY(Pad, PadProp)
-.describe(R"code(Pad an array.
+.describe(R"code(Pads an input array with a constant or edge values of the array.
 
-Only supports 4-D and 5-D input array.
+.. note:: `Pad` is deprecated. Use `pad` instead.
+
+.. note:: Current Implementation only supports 4-D or 5-D input arrays with padding applied only on the lower
```

Comment:
Implementation -> implementation
"4D or 5D" -> "4D and 5D"
"on the lower" -> "to the lower"